frontend/src/components/employees/employees.module.scss

```diff
@@ -49,3 +49,30 @@ Table {
 .removeButton {
   color: red;
 }
+
+.modalDeleteContent {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  padding: 1em;
+  background-color: white;
+}
+
+.modalDeleteButtons {
+  display: flex;
+  justify-content: flex-end;
+  & Button {
```

Comment:
when nesting SCSS I don't think we need the `&` here, or line 73. Correct me if I'm wrong plz

Reply:
You're right, they don't seem to be needed.
Removed now.